src/pyfmi/fmi.pyx

```diff
@@ -4823,26 +4823,36 @@ cdef class FMUModelBase2(ModelBase):
                 List of categories to log, call get_categories() for list of categories.
                 Default: [] (all categories)
 
-        Calls the low-level FMI function: fmi2SetDebuggLogging
+        Calls the low-level FMI function: fmi2SetDebugLogging
         """
 
         cdef FMIL.fmi2_boolean_t  log
         cdef int                  status
-        cdef FMIL.size_t          nCat = 0
-
-        self.callbacks.log_level = FMIL.jm_log_level_warning if logging_on else FMIL.jm_log_level_nothing
+        cdef FMIL.size_t          nCat = len(categories)
```

Comment:
Minor nitpick but should we rename `nCat` to `n_cat` to be more in line with the coding format?

Reply:
Fixed.